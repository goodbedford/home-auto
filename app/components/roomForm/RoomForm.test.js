/* eslint-disable no-undef */

import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import RoomForm from "./RoomForm";

//needs to test functions
describe("<RoomForm />", () => {
  // const props = {match
  const props = {
    formMsg: "add room",
    goToId: "456",
    homeId: "12",
    isLoading: false,
    isSubmitted: false,
    handleChange: jest.fn(),
    handleSubmit: jest.fn()
  };
  it("RoomForm snapshot", () => {
    const wrapper = shallow(<RoomForm {...props} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
