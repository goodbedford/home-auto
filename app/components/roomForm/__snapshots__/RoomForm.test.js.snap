// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoomForm /> RoomForm snapshot 1`] = `
<form
  className="form"
  name="roomForm"
  onSubmit={[Function]}
>
  <div
    className="form__header"
  >
    add room
  </div>
  <div
    className="form__section form__section--main"
  >
    <div
      className="form_group"
    >
      <label
        className="form__label"
      >
        <input
          className="form__input"
          maxLength="20"
          minLength="2"
          name="name"
          onChange={[Function]}
          placeholder="Ex. Dinning Room"
          required={true}
          type="text"
        />
      </label>
    </div>
  </div>
  <div
    className="controls controls--submit"
  >
    <div
      className="controls__group"
    >
      <input
        className="btn btn--submit"
        type="submit"
        value="Submit"
      />
    </div>
  </div>
</form>
`;
